packages/material-renderers/package.json

```diff
@@ -100,7 +100,7 @@
     "@jsonforms/react": "workspace:*",
     "@mui/icons-material": "^5.11.16",
     "@mui/material": "~5.13.0",
-    "@mui/x-date-pickers": "^6.5.0",
+    "@mui/x-date-pickers": "^7.7.1",
```

Comment:
Do we need to adapt the `devDependency` to `x-date-pickers` or would it also work if we keep building against the old one?

Reply:
Not needed; I can revert this back so it continues to use the old one.